tests/BUILD

```diff
@@ -1,3 +1,4 @@
+"""
```

Comment:
Why?

Reply:
The hotfix which helps us to turn off GTTA if needed make `test_core` can be passed.
This is a issue that we have met so many times. It takes a long time to recover the test. The time wasted on recovering that is not really worth it. In addition, those functions are actually covered by `driver`